src/formatting/newline_style.rs

```diff
@@ -17,6 +22,22 @@ pub(crate) fn apply_newline_style(
     }
 }
 
+/// Get the newline string based on the requested style and the original text
+pub(crate) fn get_newline_string(newline_style: NewlineStyle, raw_input_text: &str) -> String {
```

Comment:
Any reason we can't return an `&'static str` here since both `WINDOWS_NEWLINE` and `UNIX_NEWLINE` are defined as const?. Then we don't need to call `to_string()`.

Reply:
Changed.